### DISCUSSION TITLE
Difficulty in achieving “cross-border geographical coverage involving 5 MS”

### DISCUSSION THREAD

**Discussion by gmartirano** [gopeg/issues-barriers#28] [General]
Subsurface data are not generally available in national data portals. Therefore, for this use case, the multi-country geographical coverage aspect was initially addressed thanks to the stakeholder’s support, who involved the partners of existing European projects HotLime and GeoMol. These initially gave their availability to provide data, but only the Austrian partner finally provided. This was due to two reasons: 2 persons in charge in 2 MS moved to other organisations/departments and the other persons declared lack of resources to support the use case.

**Comment by thorsten-reitz** [comment]
You can download data from CH here if it helps:
https://www.swisstopo.admin.ch/de/geodata/geology/maps/geocover.html

**Comment by thorsten-reitz** [comment]
Also, there is this from FR:

https://www.brgm.fr/en/news/news/open-science-free-access-brgm-digital-geological-maps

**Comment by naomithiru** [comment]
Surface water datasets for the 5 MS are available for the Tracing use-case. However, achieving full coverage for sewer networks has so far not been achievable. The solution here has been to use the EEA Urban Waste Water Treatment Directive(UWWTD) DischargePoints dataset, which contains information on individual points of discharge from urban waste water treatment plants or collecting systems, with a spatial extent of EU27.

**Comment by AndersOstmanNovogit** [comment]
This issue is overlapping with #32 and reported in section 4.2.4 in D3.4.

**Reply by AndersOstmanNovogit** [reply]
Now in D4.3 section 4.3.3
